examples/nodejs/cache/package.json

```diff
@@ -30,6 +31,7 @@
     "typescript": "4.4.3"
   },
   "dependencies": {
+    "@aws-sdk/client-secrets-manager": "^3.370.0",
```

Comment:
Overall this PR looks great! One small detail - this "cache" example directory is supposed to be only the most basic things that people would need to get started with momento. If we add a secrets manager dep here, and people just copy this package.json file, then they might be taking on that extra secrets manager dependency unnecessarily and it can make their build a lot bigger.

So for this directory we only want to show examples that don't have any third-party dependencies.

I think we should have an `aws` examples directory that we put this into, which may contain lots of other aws examples in the future.

So you'll basically need to create a new directory and copy in the boilerplate files (package.json, tsconfig.json, .eslint*, .prettier*, etc.) and then move your script over there.